### PR TITLE
Updates for inventory type entity groups

### PR DESCRIPTION
**Pull request by jpipkin1**
## PLEASE READ

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [x] Yes, I've run `yarn install`
- [ ] No, does not apply to me

## Purpose of this pull request

This pull request updates documentation in this section:
https://help.sumologic.com/docs/cse/records-signals-entities-insights/create-an-entity-group/#create-an-entity-group-based-on-inventory-group-membership

Issue: Asana ticket: "[Field input for Entity Groups of inventory type](https://app.asana.com/0/1183741177809740/1205942116258462/f)"

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

**Comment by jpipkin1** [comment]
This PR is in draft mode during SME review (see [INVS-1811](https://sumologic.atlassian.net/browse/INVS-1811?focusedCommentId=499529)).

File: docs/cse/records-signals-entities-insights/create-an-entity-group.md [hunk 1]
```diff
@@ -92,9 +92,9 @@ Follow these instructions to create an Entity Group that corresponds to a group
 1. **Inventory Type**. Select one of:
     * Computer
     * User
-1. **Inventory Key**. Select an attribute to use from the **Inventory Type** selected above. Select **groups** if you want to use an existing Entity Group attribute. 
+1. **Inventory Key**. Select an attribute to use from the **Inventory Type** selected above. You can use second-level unnormalized inventory attributes in this field (for example, `fields.foo.bar`). Select **groups** if you want to use an existing Entity Group attribute. 
 1. **Source**. Select an inventory source from the pull-down list.
-1. **Value**. Enter a value for the attribute selected in the **Inventory Key** field above. If **groups** was selected in the **Inventory Key** field, enter the name of the group in the inventory system that contains the entities you want to add to the Entity Group. 
+1. **Value**. Enter a value for the attribute selected in the **Inventory Key** field above. You can use wildcards in this field (for example, `*OU=TCH,OU=PAR,OU=EUR*`). <br/>If **groups** was selected in the **Inventory Key** field, enter the name of the group in the inventory system that contains the entities you want to add to the Entity Group.
```
[the remark this answers is not in the document]
Comment (jpipkin1) [reply docs/cse/records-signals-entities-insights/create-an-entity-group.md:97]: Thanks for the information, @pmontiel-sumo. Using the example you provided in our Slack conversation, I changed the sentence to say:
>You can use REGEX expressions in this field (for example, [a-zA-Z]*OU=TCH,OU=PAR,OU=EUR[a-zA-Z]*).

**Review by kimsauce — APPROVED** [review]

**Comment by kimsauce** [comment]
@jpipkin1 @pmontiel-sumo approved and merging.

**Comment by jpipkin1** [comment]
Thanks, @kimsauce!

**Pull request merged**